cmd/kubeadm/app/cmd/token.go

```diff
@@ -60,22 +63,33 @@ func NewCmdToken(out io.Writer, errW io.Writer) *cobra.Command {
 		},
 	}
 
-	var token string
+	tokenCmd.PersistentFlags().StringVar(&kubeConfigFile,
+		"kubeconfig", "/etc/kubernetes/admin.conf", "The KubeConfig file to use for talking to the cluster")
+
+	var usages []string
 	var tokenDuration time.Duration
+	var description string
 	createCmd := &cobra.Command{
-		Use:   "create",
+		Use:   "create [token]",
```

Comment:
The `[token]` parameter is undefined in the help text.  I'd do `create [token-value]` and then change the description to document that "The token-value is the actual token to write.  This should be a securely generated random token of the form [a-z0-9]{6}.[a-z0-9]{16}."